packages/flutter/test/material/expand_icon_test.dart

```diff
@@ -37,7 +37,7 @@ void main() {
     );
 
     final IconTheme iconTheme = tester.firstWidget(find.byType(IconTheme));
-    expect(iconTheme.data.color, equals(Colors.black26));
+    expect(iconTheme.data.color, equals(Colors.black));
```

Comment:
This change doesn't appear to be related to the PR or the issues the PR addresses.  In fact black doesn't seem like a disabled color..

Reply:
I believe this just changed because I wrapped the app in a MaterialWidget

Reply:
The iconTheme issue appears to be related to: https://github.com/flutter/flutter/issues/3188

Reply:
According to the spec, the disabled color for icons should be black.withOpacity(0.38)

Reply:
Should I wait until this is resolved or just ignore it and let it be cleaned up by the fix?

Reply:
actually nvm, I need to make sure this is still disabled.

Reply:
Okay, the issue was that the finder was looking for the first theme. Changed to the last theme to get the disable theme. Otherwise we get the material apps default theme